.dev_scripts/github/update_copyright.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
These scripts do not belong to this pr. Maybe do a proper rebase?

Reply:
Yes, please wait a moment and I will rebase it to the latest master.